inst/css/BS4/pkgdown.css

```diff
@@ -298,40 +299,23 @@ nav[data-toggle="toc"] .nav .nav > li > a {
 /* h1 */
 
 h1.display-4 {
-  font-size: 2.5rem;
-}
-
-/* Links color contrast */
-
-a {
-  color: #0054AD;
+  font-size: $h1-font-size;
 }
 
-a:hover {
-  color: #0054AD;
-}
-
-
 /* navbar active link */
 .navbar-light .navbar-nav .active > .nav-link {
-   background: #e9ecef;
+   background: $secondary;
    border-radius: 1rem;
  }
 .navbar-nav .nav-link {
  padding-left: 0.5rem;
  }
 
-@include media-breakpoint-up(md) {
-  .navbar-nav .nav-link {
```

Comment:
with the current rounded form there should always be padding on the left, otherwise it looks bad.